### PR TITLE
Dev

### LINKED ISSUE #1: Initial push

### LINKED ISSUE #2: Create homepage

### LINKED ISSUE #3: Create login

### LINKED ISSUE #4: Create profile

### LINKED ISSUE #6: Fix contrast errors white on green/green on white
Logo as well as buttons have contrast errors which need fixing

### LINKED ISSUE #7: Check custom.scss for styling which could be done through Bootstrap

### LINKED ISSUE #8: Fix profile image overlapping profile info below 500px

### LINKED ISSUE #9: Add list of user friends on profile

### LINKED ISSUE #10: Make posts on homepage smaller

### PR DESCRIPTION
**Pull request by pretzL**

### CODE REVIEW

File: login.html [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+                      <input type="checkbox" checked="checked" />
+                      <div class="control__indicator"></div>
+                    </label>
+                    <span class="ml-auto"><a href="#" class="forgot-pass login-a">Forgot Password</a></span>
+                  </div>
+                  <div class="d-flex justify-content-between">
+                    <input type="submit" value="Log In" class="btn btn-block bg-color-green" />
+                    <a href="/register.html" class="btn btn-block bg-color-green">Register</a>
+                  </div>
+                </form>
```
Comment (jaaneh) [inline login.html]: Your form action attribute should be set to `/profile` like the assignment brief mentions.

You should perform some client-side validation in your form. Things like `required`, `minlength`, `pattern` attributes. See [MDN Docs](https://developer.mozilla.org/en-US/docs/Learn/Forms/Form_validation).

File: profile.html [hunk 2]
```diff
@@ -0,0 +1,229 @@
...
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta name="keywords" content="Chattes, Social Media, Bootstrap, SASS, Noroff" />
+    <meta name="description" content="Vi Chattes!" />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link href="https://fonts.googleapis.com/css2?family=Pacifico&display=swap" rel="stylesheet" />
+    <link rel="stylesheet" href="dist/css/styles.css" />
+    <script defer src="/node_modules/bootstrap/dist/js/bootstrap.bundle.min.js"></script>
+    <script defer src="js/index.js"></script>
```
Comment (jaaneh) [inline profile.html:14]: The links to the script files do not function. Your `js` folder is actually named `JS` with capitalized letters. Make sure these match in all locations. Perhaps easiest to just rename your JS folder and fix any places you may have used the correct src.

File: index.html [hunk 3]
```diff
@@ -4,7 +4,216 @@
...
+    <title>Chattes</title>
   </head>
-  <body></body>
+  <body>
+    <header class="py-2 border-bottom">
+      <div class="container d-flex flex-wrap justify-content-between container-xxl">
+        <a href="/index.html" class="d-flex align-items-start mb-3 mb-lg-0 me-lg-auto text-decoration-none hover-green color-green">
+          <h1 class="">Chattes</h1>
+        </a>
+        <div class="btn-group hover-green">
```
Comment (jaaneh) [inline index.html:23]: Clicking this search icon doesn't actually bring up any search field. Perhaps you can display the input field at all times? Are you wanting to implement the show/hide with JavaScript?

File: login.html (hunk 1 above)
Comment (pretzL) [reply login.html]: Resolved by [de42907](https://github.com/pretzL/chattes/pull/11/commits/de429070b4d5b79e16ccca1391c0a00a6ba022d5)

File: profile.html (hunk 2 above)
Comment (pretzL) [reply profile.html:14]: Resolved by [de42907](https://github.com/pretzL/chattes/pull/11/commits/de429070b4d5b79e16ccca1391c0a00a6ba022d5)

File: index.html (hunk 3 above)
Comment (jaaneh) [reply index.html:23]: This is not actually a problem. This is resolved

**Comment by jaaneh** [comment]
Looks good to me. Thanks for fixing the little issues I found :-)

**Pull request merged**